### PR TITLE
Added configmap controller service for automatic restarts after config map changes

### PR DESCRIPTION
**Pull request by quintesse**
PD: not sure what the "hash" value should be for something that doesn't get built and is always the same

### CODE REVIEW

File: launchpad-services/configmapcontroller.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+services:
+- hash: db0d424c677142d4f152075f4f968e8c990881a6
```
Comment (gastaldi) [inline launchpad-services/configmapcontroller.yaml]: This should be 971412133b88b47660c33cb77a6ec1fd4a392558

Comment (quintesse) [reply launchpad-services/configmapcontroller.yaml]: why?

Comment (gastaldi) [reply launchpad-services/configmapcontroller.yaml]: As in https://github.com/fabric8-launcher/launcher-openshift-templates/commit/971412133b88b47660c33cb77a6ec1fd4a392558

Comment (quintesse) [reply launchpad-services/configmapcontroller.yaml]: ah because it's the hash it will use to check out that file? ok

Comment (gastaldi) [reply launchpad-services/configmapcontroller.yaml]: Just guessing, as that is the latest commit ID in the repo. The Hash you provided doesn't seem to have the file in the new repository name change path

**Comment by gastaldi** [comment]
Closing as this PR is stale for a while and it's safer to let the SD team do the restarts

**Pull request closed without merging**